cmd/crossplane/core/init.go

```diff
@@ -97,6 +92,17 @@ func (c *initCommand) Run(s *runtime.Scheme, log logging.Logger) error {
 		)
 	}
 
+	// CRD migrator steps are done after core CRDs are applied/updated, so we
+	// are always migrating to the most current storage version
+	steps = append(steps,
+		initializer.NewCoreCRDsMigrator("compositionrevisions.apiextensions.crossplane.io", "v1alpha1"),
+		initializer.NewCoreCRDsMigrator("environmentconfigs.apiextensions.crossplane.io", "v1beta1"),
+		initializer.NewCoreCRDsMigrator("usages.apiextensions.crossplane.io", "v1beta1"),
```

Comment:
also note that this Usages migrator was added within this moved migrator steps block. https://github.com/crossplane/crossplane/pull/6226/commits/773c40947e0bfbc769f98e7470dbc12899f96a4e shows that addition in isolation.